### DISCUSSION TITLE
配置reCAPTCHA后评论报错 || Report error after configuring reCAPTCHA

### DISCUSSION THREAD

**Discussion by 2176608** [walinejs/waline#2021] [Q&A] [answered]
### Describe the bug | 问题描述

配置reCAPTCHA后评论报错，但是用管理员身份发表评论却可以
![image](https://github.com/walinejs/waline/assets/90504717/5b23a126-b5c2-4117-a978-6d329cb4edc6)

![Snipaste_2023-07-13_23-10-19](https://github.com/walinejs/waline/assets/90504717/6f4e6cba-83ae-4947-b40b-f05858a28ab0)

### Website URL | 问题网站

https://2176608.github.io

### Where your waline deploy? | 服务部署在哪里？

Deta

### Where your comment data store? | 数据存储在哪里？

GitHub

---
### Describe the bug | Problem description

After configuring reCAPTCHA, the comment reports an error, but it is possible to post a comment as an administrator
![image](https://github.com/walinejs/waline/assets/90504717/5b23a126-b5c2-4117-a978-6d329cb4edc6)

![Snipaste_2023-07-13_23-10-19](https://github.com/walinejs/waline/assets/90504717/6f4e6cba-83ae-4947-b40b-f05858a28ab0)

### Website URL | Problem Website

https://2176608.github.io

### Where your waline deploy? | Where is the service deployed?

Deta

### Where your comment data store? | Where is your comment data store?

GitHub

**Comment by lizheming** [comment]
reCAPTCHA 前端需要配置 reCAPTCHA 的 key，是不是没有配置

---
The reCAPTCHA front-end needs to configure the reCAPTCHA key, is it not configured

**Reply by 2176608** [reply]
配置了，其他的地方也配置好了

---
Configured, other places are also configured

**Reply by lizheming** [reply, marked as answer]
配置有问题，自己解决下
![image](https://github.com/walinejs/waline/assets/424491/6b9f00c2-02a5-45fe-bf2e-bbba1871f554)

**Reply by 2176608** [reply]
好的，感谢回答！！！

---
Ok, thanks for answering! ! !
